### ISSUE TITLE
"NumPad +" won't assign

### ISSUE THREAD

**Issue by nzxt** [Dafrok/v-hotkey#6]
Can't assign `Numpad +` as hotkey, but `Numpad -` works like a charm.
```
      keymap () {
        return {
          'numpad -': this.volumeDown,
          'numpad +': this.volumeUp
        }
      }
```
Where to dig in? Thanks!

**Comment by BerezaRostislav** [comment]
i'm having a same issue

**Comment by Dafrok** [comment]
I doubt it's a problem caused by wrong key code.
Try putting the line below into your devtool, and then press the `numpad +`, tell me that key code in console.

```js
document.addEventListener('keypress', e => console.log(e.keyCode))
```

**Comment by nzxt** [comment]
Yes, it says numpad+ has keycode '43', like generic '+'.
How to overide 'numpad+' keycode or use code numbers like '43 'instead of 'numpad +'?
And how to assign generic '+', like 'alt+ +' or not?

**Comment by Dafrok** [comment]
Just bind your handler handler to a generic `+` key should resolve your problem. That `numpad +` works for some elder keyboards I guess.

**Comment by Dafrok** [comment]
Just bind your handler handler to a generic + key should resolve your problem. That numpad + works for some elder keyboards I guess.

**Comment by nzxt** [comment]
`numslock` does not affects. `numpad -` work well in any conditions.
Please, show on how to assign generic `+` with `alt`, for example
```
keymap () {
    return {
        'alt+ +': this.volumeUp
    }
}
```
or not? Or maybe add `plus` like keyword?

**Comment by Dafrok** [comment]
That's really a problem as I forgot the `+` is a keynote and it is also an operator. So I published a new version which modified the key map just now. Now you can use `alt++` / `alt + +` / `alt + add` to resolve your problem.

**Comment by nzxt** [comment]
Great thanks! I'll try a bit later. Kewl module. I'm lovin' it!

**Comment by Dafrok** [comment]
I have no keyboards with a numpad, so I can't catch the problem. But, I found there is something different between`keypress` and `keydown`.

Try this code and tell me the result:

```
document.addEventListener('keydown', e => console.log(e.keyCode))
```

**Comment by Dafrok** [comment]
Try `alt + numpad +` in version 0.2.2, I took a few change

**Comment by nzxt** [comment]
Tried `alt+numpad + / alt + numpad add / numpad + / numpad add` but with no luck. Can't figure it out too.. 
`keydown` says `keycode '+' = 43`, `keycode '-' = 45`, all seems to be ok.
Thanks for your participation ;)

**Comment by Dafrok** [comment]
wft,

**Comment by Dafrok** [comment]
I have published version `0.2.3`, I hope that is my last try for this

**Comment by nzxt** [comment]
Sorry, but no luck w/ this bug %)

**Comment by Dafrok** [comment]
Maybe resolved in #14
